dev/NavigationView/NavigationView.cpp

```diff
@@ -1930,13 +1936,24 @@ void NavigationView::ChangeSelection(const winrt::IInspectable& prevItem, const
         UnselectPrevItem(prevItem, nextItem);
         ChangeSelectStatusForItem(nextItem, true /*selected*/);
 
-        if (winrt::AutomationPeer peer = winrt::FrameworkElementAutomationPeer::FromElement(*this))
+        // Selection changed and we need to notify UIA
+        // HOWEVER expand collapse can also trigger if an item can expand/collapse
+        // There are multiple cases when selectino changes:
```

Comment:
While "selectino" does sound rather cool, I think we should stick with "selection" here 😁

Reply:
Haha good point, updated it.